src/test/test-time-util.c

```diff
@@ -648,7 +648,7 @@ static void test_parse_timestamp_one(const char *str, usec_t max_diff, usec_t ex
         int r;
 
         r = parse_timestamp(str, &usec);
-        log_debug("/* %s(%s): max_diff="USEC_FMT", expected="USEC_FMT", result="USEC_FMT" */", __func__, str, max_diff, expected, usec);
+        fprintf(stderr, "/* %s(%s): max_diff="USEC_FMT", expected="USEC_FMT", result="USEC_FMT" */\n", __func__, str, max_diff, expected, usec);
```

Comment:
Why change this? We have `SYSTEMD_LOG_DEBUG` to access these.

Reply:
Yep, included by accident with commit -a; removed